doc/introduction/interfaces.rst

```diff
@@ -3,17 +3,99 @@
 
 .. _intro_interfaces:
 
-Interfaces and training
-=======================
+Gradients and training
+======================
 
 PennyLane offers seamless integration between classical and quantum computations. Code up quantum
-circuits in PennyLane, and connect them easily to the top scientific computing and machine learning
-libraries.
+circuits in PennyLane, compute :doc:`gradients of quantum circuits <glossary/quantum_gradient>`, and
+connect them easily to the top scientific computing and machine learning libraries.
+
+Gradients
+---------
+
+When creating a QNode, you can specify the :doc:`differentiation method
+<glossary/quantum_differentiable_programming>` that PennyLane should use whenever the gradient of
+that QNode is requested.
+
+.. code-block:: python
+
+    @qml.qnode(dev, diff_method="parameter-shift")
+    def circuit(x):
+        qml.RX(x, wires=0)
+        return qml.probs(wires=0)
+
+PennyLane currently provides the following differentiation methods for QNodes, divided into
+the following three categories.
```

Comment:
Maybe reword this so that "following" is only used once. Something like: "PennyLane provides a few different differentiation methods for QNodes, divided into the following categories."